cogito/gchatsink.go

```diff
@@ -22,9 +22,14 @@ func (sink GoogleChatSink) Send() error {
 	sink.Log.Debug("send: started")
 	defer sink.Log.Debug("send: finished")
 
-	if sink.Request.Source.GChatWebHook == "" {
-		sink.Log.Debug("not sending to chat",
-			"reason", "feature not enabled")
+	// If present, params.gchat_webhook overrides source.gchat_webhook.
+	webHook := sink.Request.Source.GChatWebHook
+	if sink.Request.Params.GChatWebHook != "" {
+		webHook = sink.Request.Params.GChatWebHook
+		sink.Log.Debug("params.gchat_webhook is overriding source.gchat_webhook")
+	}
+	if webHook == "" {
+		sink.Log.Info("not sending to chat", "reason", "feature not enabled")
```

Comment:
I transformed the log level of this one from debug to info (thus default) because I feel that it will reduce confusion.